GodotAddinVS/source.extension.vsixmanifest

```diff
@@ -1,15 +1,32 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="GodotAddinVS" Version="1.1.1" Language="en-US" Publisher="Ignacio Roldán Etcheverry" />
+        <Identity Id="GodotAddinVS" Version="1.1.2" Language="en-US" Publisher="Ignacio Roldán Etcheverry" />
         <DisplayName>Godot Support</DisplayName>
         <Description xml:space="preserve">Support for Godot Engine C# projects, including debugging and extended code completion.</Description>
         <License>LICENSE.txt</License>
         <Icon>icon.png</Icon>
         <Tags>Godot</Tags>
     </Metadata>
     <Installation>
-        <InstallationTarget Version="[16.0,17.0)" Id="Microsoft.VisualStudio.Community" />
+        <InstallationTarget Version="[16.0,18.0)" Id="Microsoft.VisualStudio.Community">
+            <ProductArchitecture>amd64</ProductArchitecture>
+        </InstallationTarget>
+        <InstallationTarget Version="[16.0,18.0)" Id="Microsoft.VisualStudio.Pro">
+            <ProductArchitecture>amd64</ProductArchitecture>
+        </InstallationTarget>
+        <InstallationTarget Version="[16.0,18.0)" Id="Microsoft.VisualStudio.Enterprise">
+            <ProductArchitecture>amd64</ProductArchitecture>
+        </InstallationTarget>
+        <InstallationTarget Version="[16.0,18.0)" Id="Microsoft.VisualStudio.Community">
+            <ProductArchitecture>x86</ProductArchitecture>
+        </InstallationTarget>
+        <InstallationTarget Version="[16.0,18.0)" Id="Microsoft.VisualStudio.Pro">
+            <ProductArchitecture>x86</ProductArchitecture>
+        </InstallationTarget>
+        <InstallationTarget Version="[16.0,18.0)" Id="Microsoft.VisualStudio.Enterprise">
+            <ProductArchitecture>x86</ProductArchitecture>
```

Comment:
The `ProductArchitecture` nodes cause a build error on VS19. It's not recognized there. Do we need it?

Reply:
No, we don't need it. I am working on a solution, which just seperates the VS22 version as 2.x and VS19 version as 1.x, because it would be easier to work with short term. I don't know, how much attention this extension will get, so it would be a quick, not ideal, but practical fix.

Reply:
Interesting. If I remove the cap, I can build with both VS19 and 22, but it still only installs on VS19 only, with `Microsoft.VSSDK.BuildTools` set to `16.11.35` and `Microsoft.VisualStudio.SDK` set to `16.10.31321.278` (kinda the last versions VS19 works with). I assume VS22 works with only `Microsoft.VisualStudio.SDK 17.x` and `Microsoft.VSSDK.BuildTools 17.x`.